prov/efa/src/efa_prov_info.c

```diff
@@ -88,29 +94,29 @@ const struct fi_domain_attr efa_domain_attr = {
  *
  * @param 	prov_info[out]		pointer to prov_info object
  * @param	device[in]		pointer to an efa_device struct, which contains device attributes
- * @param	ep_type[in]		endpoint type, can be FI_EP_RDM or FI_EP_DGRAM
+ * @param	info_type[in]		EFA info type
  * @return	0 on sucessess
  * 		negative libfabric error code on failure
  */
 static
 int efa_prov_info_set_domain_attr(struct fi_info *prov_info,
 				  struct efa_device *device,
-				  enum fi_ep_type ep_type)
+				  enum efa_info_type info_type)
 {
 	size_t name_len;
 
 	*prov_info->domain_attr = efa_domain_attr;
 	prov_info->domain_attr->av_type = FI_AV_UNSPEC;
 
 	/* set domain name */
-	name_len = strlen(device->ibv_ctx->device->name) + strlen(efa_domain_name_suffix(ep_type));
+	name_len = strlen(device->ibv_ctx->device->name) + strlen(efa_domain_name_suffix(info_type));
 	prov_info->domain_attr->name = malloc(name_len + 1);
 	if (!prov_info->domain_attr->name) {
 		return -FI_ENOMEM;
 	}
 
 	snprintf(prov_info->domain_attr->name, name_len + 1, "%s%s",
-		 device->ibv_ctx->device->name, efa_domain_name_suffix(ep_type));
+		 device->ibv_ctx->device->name, efa_domain_name_suffix(info_type));
```

Comment:
For `	prov_info->domain_attr->resource_mgmt	= FI_RM_DISABLED;` , I think we need to make it as `FI_RM_ENABLED` for efa-direct, as we set infinite rnr retry for the qp which is some kind of resource management.. The current fi_rma_bw requires `FI_RM_ENABLED` so if we leave it as disabled it will make fi_getinfo failed.